### DISCUSSION TITLE
Zksync Local Deploy video

### DISCUSSION THREAD

**Discussion by 48cork** [Cyfrin/foundry-full-course-cu#2595] [General]
what in this command Patrick is using  the RPC-URL=http://127.0.0.1:8011 and not anymore  the RPC-URL=http://127.0.0.1:8545?

thanks in advance

**Comment by EngrPips** [comment]
Hello @48cork, I don't understand your question.

**Comment by Napcalx** [comment]
The Port number he is using changes from 8011 to 8545, when you run anvil it produces the Url that has the port number of 8545 in it, so probably there is a new location from which he is getting the RPC Url

**Comment by EngrPips** [comment]
This `RPC-URL=http://127.0.0.1:8011/` is for local zksync chain while this `RPC-URL=http://127.0.0.1:8545/` is for local Anvil chain.

**Reply by 48cork** [reply]
thanks

**Reply by EngrPips** [reply]
You are welcome, Friend.
